modules/upgrade/pages/migrate-3x-4x.adoc

```diff
@@ -1,3 +1,445 @@
+IMPORTANT: During the migration process, the target system will fake its host name to be the same as the source system, this is necessary as the host name of a {productname} installation is vital and should not be changed once set.
+Therefore do not be confused when logging in to your systems during migration; they both will present you with the same host name.
++
+
+. Continue by following the normal installation steps.
++
+.Database Parameters
+IMPORTANT: Specify the database parameters using the same database parameters as the source system is recommended.
+At least, using the the same database credentials as when creating the source or original {productname} database is mandatory.
+
+. After all the data has been gathered, {yast} will terminate.
+
+
+The actual migration will not start automatically but needs to be triggered manually as outlined in <<bp.sec.mgr.migration.performing>>.
+
+
+
+[[bp.sec.mgr.migration.performing]]
+== Performing the Migration
+
+The migration is performed by excecuting the following command:
+
+----
+/usr/lib/susemanager/bin/mgr-setup -m
+----
+
+This command reads the data gathered during <<proc.mgr.migration.setup.target>>, sets up {productname} onto a new target machine and transfers all of the data from the source machine.
+As several operations need to be performed on the source machine via SSH, you will be prompted once for the root password of the source machine.
+A temporary SSH key named `migration-key` is created and installed on the source machine, so you need to give the {rootuser} password only once.
+The temporary SSH key will be deleted after successful migration.
+Ideally, this is all you will need to do.
+
+Depending on the size of the installation, the actual migration will take up to several hours.
+When successfully finished, you will be notified ("_Migration complete._") and prompted to shut down the source machine, re-configure the network of the target machine to use the same IP address and host name as the original machine and restart it.
+It should now be a fully functional replacement for your previous {productname} 3.2  installation.
+
+The following numbers illustrate the runtime for dumping and importing the database:
+
+----
+14:53:37   Dumping remote database to /var/spacewalk/tmp/susemanager.dmp.gz on target system. Please wait...
+14:58:14   Database successfully dumped. Size is: 506M
+14:58:29   Importing database dump. Please wait...
+15:05:11   Database dump successfully imported.
+----
+
+
+For this example dumping the database takes around five minutes to complete.
+Importing the dump onto the target system will take an additional seven minutes.
+For big installations this can take up to several hours.
+You should also account for the time it takes to copy all the package data to the new machine.
+Depending on your network infrastructure and hardware, this can also take a significant amount of time.
+
+
+
+[[bp.sec.mgr.migration.speedup]]
+== Speeding up the Migration
```

Comment:
I don't think there needs to be a separate heading here, it directly relates to the previous information.

Reply:
addressed separately